### ISSUE TITLE
Create an "Awesome TSDX" repo

### ISSUE THREAD

**Issue by kylemh** [jaredpalmer/tsdx#460]
I thought it'd be cool for there to be space espousing and showcasing the power that TSDX has given them.

**Comment by sw-yx** [comment]
🤔 
![image](https://user-images.githubusercontent.com/6764957/73412314-3f42d300-42d6-11ea-9492-ff6ceda52a03.png)

**Comment by kylemh** [comment]
Right, but there's no filtering here... There's no seeing if it's a library that I might be able to use and can attribute partial success to `tsdx`

Prime example... I could check the `used by` for ESLint, but that doesn't get me this:
https://github.com/dustinspecker/awesome-eslint

**Comment by sw-yx** [comment]
up to you, if you wanna maintain it we can link from readme!
